src/AlertFlags/AlertFlags.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './AlertFlags.scss'
+
+const AlertFlags = ({ alerts, category }) => {
+  function isShown(code) {
+    if (alerts) {
+      return alerts.some(alert => {
+        if (alert.get) {
+          if (!alert.get('expired')) {
```

Comment:
unnecessary nested block, would be better as
```javascript
if (!alert.get('expired') &&alert.get('alertCode') === code) return true
```

Reply:
As discussed, `isShown` is existing code just shows as new because of file rename.